### DISCUSSION TITLE
OnlyT 2.1.0.1

### DISCUSSION THREAD

**Discussion by AntonyCorbett** [AntonyCorbett/OnlyT#481] [Announcements]
Optional "minimise window" behaviour that reduces window size.
Updated Nuget packages.

<hr /><em>This discussion was created from the release <a href='https://github.com/AntonyCorbett/OnlyT/releases/tag/2.1.0.1'>OnlyT 2.1.0.1</a>.</em>

**Comment by AntonyCorbett** [comment]
You can now resize the main window so that it is much smaller than previously possible. Some of the controls automatically disappear as the size is reduced. You can also use the window's Minimise button to reduce the window to a small size (this optional behaviour is turned on in Settings by default).

**Reply by zcabrera1** [reply]
Hi, I'm having an issue where after downloading the app and installing  I get several error messages,  the app will eventually open and work but the message comes back randomly.  Any suggestions?

**Reply by AntonyCorbett** [reply]
Please tell me what the error messages are. You could also send me the log file

**Comment by Syntax-Prime** [comment]
Hello,

I'm encountering an issue whereby the control disappears when I enable the display to a secondary monitor. I am not then able to start the timer. I cannot close the app too. I have to access the process manager to do so. Your help is very much appreciated.

Thank you so much, brother.

**Reply by AntonyCorbett** [reply]
Do you mean that the timer window is displayed on the same monitor as the OnlyT main window, thus covering it?

**Comment by Syntax-Prime** [comment]
Hello Brother Anthony,

Thank you for your prompt response. No, the control just disappears. I am using three displays. If I click on the button on top at the settings window to go back to the normal view, it’s just the time that appears but no way to start the time. I also tried using the “Windowed” mode thinking that the timer is covering the controls. I tried moving it around but I do not see it. I am using Windows 11 if that helps. Thank you again for your assistance, brother.

**Reply by AntonyCorbett** [reply]
Thanks for the feedback. Is the app minimised? There should be 2 OnlyT icons on the taskbar representing the main window and the timer window. If so, you should be able to click each icon to toggle between OnlyT's 2 windows.

**Comment by Syntax-Prime** [comment]
Hello Brother Antony,

There is only one icon on my taskbar. Please see the screenshot below.

***@***.***

The app is not minimized. The interface shown above is the default when I launch it. The whole interface also disappears when I click on the expand button. So I cannot also change the time if I am to set it manually. It is stuck at 30:00. What I do is I have to manually end the task using the Task Manager. Then relaunch the app so I can use it but only if it is for 30 minutes like for Public Talks. Thank you so much again for the assistance, brother.

**Comment by AntonyCorbett** [comment]
Please would you raise an Issue in the Issues page? Also supply a copy of your log file (from Documents\OnlyT\logs).

**Comment by Syntax-Prime** [comment]
Dear Brother Antony,

I did raise an issue and also sent my log files.

Thank you so much for your assistance.
